pwndbg/gdblib/qemu.py

```diff
@@ -107,4 +109,6 @@ def pid():
 
         for c in connections:
             if c.laddr in targets:
-                return process.pid
+                pid = process.pid
+                break
+    return pid
```

Comment:
There are two loops so the 'break' breaks only from the first one. The previous return was ok

Reply:
fixed it